core/controllers/creator_dashboard.py

```diff
@@ -346,6 +337,7 @@ def post(self):
         creator_dashboard_display_pref = self.payload.get('display_preference')
         user_services.update_user_creator_dashboard_display(
             self.user_id, creator_dashboard_display_pref)
+        self.render_json({})
```

Comment:
If we don't add this, the response type by doing a post_json will be text/html and the assertion would fail [here](https://github.com/oppia/oppia/blob/develop/core/tests/test_utils.py#L686)